src/treemodel/node.js

```diff
@@ -188,6 +188,9 @@ export default class Node {
 		// Due to circular references we need to remove parent reference.
 		json.parent = this.parent ? this.parent.name : null;
 
+		// Serialize attributes as Map object is represented as "{}" when parsing to JSON.
+		json._attrs = [ ...json._attrs ];
```

Comment:
That's some point but we have this in more places:
```js
let node = new Element( 'one' );

expect( treeModelTestUtils.jsonParseStringify( node ) ).to.deep.equal( {
	_attrs: [],
	_children: {
		_indexMap: [],
		_nodes: []
	},
	name: 'one'
} );
```

So for consistency I've left `_attrs` but the question about all private properties remains. It would be nice of you (@reinmar, @pjasiun) to give me your definitive view on this: change them or not?